bokeh/models/annotations.py

```diff
@@ -40,10 +40,16 @@ class Legend(Annotation):
     The %s for the legend background style.
     """)
 
+    background_fill_color = Override(default="#ffffff")
+
     label_props = Include(TextProps, help="""
     The %s for the legend labels.
     """)
 
+    label_text_baseline = Override(default='middle')
+
+    label_text_font_size = Override(default={ 'value' : '10pt' })
+
```

Comment:
Why are these `Overrides`? AFAIK `Legend` inherits from `Annotation` inherits from `Renderer` neither of which define these props?